gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -333,23 +357,12 @@ def get_predetermined_qc(version: str = CURRENT_VERSION, test: bool = False):
     },
 )
 
-# Samples to drop for PCA due to them being related
-pca_related_samples_to_drop = VersionedTableResource(
+# Samples to drop due to them being related
+related_samples_to_drop = VersionedTableResource(
     CURRENT_VERSION,
     {
         version: TableResource(
-            f"{get_sample_qc_root(version)}/gnomad.exomes.v{version}.related_samples_to_drop_for_pca.ht"
-        )
-        for version in VERSIONS
-    },
-)
-
-# Related samples to drop for release
```

Comment:
We will actually still need both. One is used for the population PCA and the other is used for release and includes an option to prioritize samples that pass outlier filtering which happens after population inference

Reply:
Thanks, reverted this. I keep forgetting that we should be able to run the PCA (even though PC-Relate would have been problematic).